### PR TITLE
feat: Add GKE Autopilot blueprint

### PR DESCRIPTION
**Pull request by kaariger**
- a blueprint to create GKE Autopilot clusters
- added attribution
- added release please config
- updated README

### CODE REVIEW

File: catalog/gke-autopilot/Kptfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: kpt.dev/v1
```
Comment (morgante) [inline catalog/gke-autopilot/Kptfile:1]: This blueprint should be derivative of the existing GKE blueprint (ie. upstream should be GKE). You first step for build this blueprint should have been to run `kpt pkg get`.

**Review by morgante — CHANGES_REQUESTED** [review]

File: catalog/gke-autopilot/Kptfile (hunk 1 above)
Comment (kaariger) [reply catalog/gke-autopilot/Kptfile:1]: That is how I started but I removed the upstream attribution to make it standalone. I will add that back in.

Comment (kaariger) [reply catalog/gke-autopilot/Kptfile:1]: Kpt file updated

Comment (morgante) [reply catalog/gke-autopilot/Kptfile:1]: Upstream is still not present. Please push!

Comment (kaariger) [reply catalog/gke-autopilot/Kptfile:1]: Pushed the changes

**Review by morgante — APPROVED** [review]

**Pull request merged**